plugins/talk-plugin-local-auth/client/components/ChangeEmailContentDialog.js

```diff
@@ -86,14 +157,11 @@ class ChangeEmailContentDialog extends React.Component {
 }
 
 ChangeEmailContentDialog.propTypes = {
-  save: PropTypes.func,
   next: PropTypes.func,
-  cancel: PropTypes.func,
-  onChange: PropTypes.func,
+  save: PropTypes.func,
   formData: PropTypes.object,
   email: PropTypes.string,
-  hasError: PropTypes.func,
-  getError: PropTypes.func,
+  closeDialog: PropTypes.func,
```

Comment:
Something I'm going to suggest more in the future and in the arch meetings, that   I'd name these properties `onCloseDialog` instead to emphasis that this is a sort of event just like HTML components does with `onClick`, `onCancel`. Moving away from _I'm passing in a fuction_ to  _I'm reacting to a event_ which puts an emphasis on isolation.